src/UserManager.ts

```diff
@@ -4,6 +4,14 @@ import { ServiceClass } from './ServiceClass';
 import { AxiosInstance } from 'axios';
 import { AuthHelper } from './util/AuthenticationHelper'
 
+declare function require(name: any): any;
+
+// Node specific storage
+if (typeof localStorage === "undefined" || localStorage === null) {
+    var LocalStorage = require('node-localstorage').LocalStorage;
+    var localStorage = new LocalStorage('/tmp/test');
```

Comment:
Could we rename the lowercase localstorage? Differing by one letter could be pretty confusing.

Reply:
lower case localstorage is normally a standard part of the web browser API, but we could change LocalStorage to something else or just not even use it, though the line would be long